### PR TITLE
fix: Hackathon styling and resource update

### PR DESCRIPTION
**Pull request by jeremytchang**
- Updated Loading component to default to always display if not
passed in loading prop. Making it as dumb as possible
- Updated Loading rendering logic and styling in scene components
so layouts does not shift/change after loading is finished.
- Updated side nav to be thinner to accomodate smaller viewports
- Updated resources with new component and dataset tag
- Added header to each page for easier navigability
- Added hacker faq link to home/agenda page
- Added additional text and field styling to resource page for
better UX

### CODE REVIEW

File: packages/hackathon/src/scenes/HomeScene/HomeScene.tsx [hunk 1]
```diff
@@ -42,9 +43,18 @@ export const HomeScene: FC<HomeSceneProps> = ({ hacker }) => {
   return (
     <>
       <SpaceVertical gap="u5">
-        <Heading as="h2" fontSize="xxxlarge" fontWeight="medium">
-          Agenda
-        </Heading>
+        <Span>
```
Comment (jeremytchang) [inline packages/hackathon/src/scenes/HomeScene/HomeScene.tsx:46]: Nested in span so the paragraph is not too far from Heading in SpaceVertical.

Comment (jkaster) [reply packages/hackathon/src/scenes/HomeScene/HomeScene.tsx:46]: For v.next with artifact storage we'll schedule some design review too

**Review by jkaster — APPROVED** [review]
Lots of nice little improvements. Good sweep.

File: packages/hackathon/src/scenes/HomeScene/HomeScene.tsx (hunk 1 above)
Comment (jeremytchang) [reply packages/hackathon/src/scenes/HomeScene/HomeScene.tsx:46]: Totally. I'm pretty sure I'm not following design best practices here, only aiming for what looks good. Will appreciate their input.

**Pull request merged**